xsd/netex_part_1/part1_ifopt/netex_ifopt_path_version.xsd

```diff
@@ -191,7 +191,7 @@ Rail transport, Roads and Road transport
 					<xsd:documentation>Nature of the  tactile warning strips (in the direction of the PATH LINK). +v1.1</xsd:documentation>
 				</xsd:annotation>
 			</xsd:element>
-			<xsd:element name="TactileWarningStripContrast" type="xsd:boolean" minOccurs="0">
+			<xsd:element name="TactileWarningStripContrast" type="xsd:boolean" minOccurs="0" default="true">
```

Comment:
So this would mean that there is always contrast?

Reply:
Not really, it is just to say that the contrast is good by default and you need to be explicit if not... just a proposal here